client/me/purchases/index.js

```diff
@@ -15,37 +15,17 @@ import paths from './paths';
 
 export default function() {
 	if ( config.isEnabled( 'manage/payment-methods' ) ) {
-		page(
-			paths.addCreditCard(),
-			meController.sidebar,
-			controller.addCreditCard
-		);
+		page( paths.addCreditCard(), meController.sidebar, controller.addCreditCard );
 
 		// redirect legacy urls
-		page(
-			'/payment-methods/add-credit-card',
-			() => page.redirect( paths.addCreditCard() )
-		);
+		page( '/payment-methods/add-credit-card', () => page.redirect( paths.addCreditCard() ) );
 	}
 
-	page(
-		paths.billingHistory(),
-		meController.sidebar,
-		billingController.billingHistory
-	);
+	page( paths.billingHistory(), meController.sidebar, billingController.billingHistory );
 
-	page(
-		paths.billingHistoryReceipt(),
-		meController.sidebar,
-		billingController.transaction
-	);
+	page( paths.billingHistoryReceipt(), meController.sidebar, billingController.transaction );
 
-	page(
-		paths.purchasesRoot(),
-		meController.sidebar,
-		controller.noSitesMessage,
-		controller.list
-	);
+	page( paths.purchasesRoot(), meController.sidebar, controller.noSitesMessage, controller.list );
```

Comment:
Another good example of changes that I don't like at all: we're now mixing short and long `page()` calls in this file, which makes just them harder to scan.